pkg/datapath/linux/config/config.go

```diff
@@ -31,6 +31,8 @@ import (
 	"github.com/cilium/cilium/pkg/defaults"
 	"github.com/cilium/cilium/pkg/identity"
```

Comment:
If we opt by me comment left in the previous commit this commit can be dropped.

Reply:
That's a good idea, I will do as you suggest and drop this commit.